Telemetry/RulesTest.cs

```diff
@@ -25,11 +47,335 @@ public void Should_Return_OK_Status()
+            var ruleRequest = this.GetSampleRuleWithCalculation("Average", "600000");
+
+            var request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX + "/" + newRuleId);
+            request.AddHeader("Content-Type", "application/json");
+            request.SetContent(JsonConvert.SerializeObject(ruleRequest));
+
+            var newRuleResponse = this.httpClient.PutAsync(request).Result;
+            var newRule = JsonConvert.DeserializeObject<RuleApiModel>(newRuleResponse.Content);
+
+            // Dispose after tests run
+            this.disposeRulesList.Add(newRule.Id);
+
+            // Act
+            newRule.Enabled = false;
+
+            request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX + "/" + newRule.Id);
+            request.AddHeader("Content-Type", "application/json");
+            request.SetContent(JsonConvert.SerializeObject(newRule));
+
+            var updateResponse = this.httpClient.PutAsync(request).Result;
+            var updatedRule = JsonConvert.DeserializeObject<RuleApiModel>(updateResponse.Content);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, updateResponse.StatusCode);
+            Assert.Equal(newRule.Enabled, updatedRule.Enabled);
+        }
+
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void DeleteRuleReturnsOK_IfRuleExists()
+        {
+            string ruleId = "TESTRULEID" + DateTime.UtcNow.ToString("yyyyMMddHHmmss") + "-" + Guid.NewGuid();
+
+            // Arrange  
+            var ruleRequest = this.GetSampleRuleWithCalculation("Average", "600000");
+
+            var request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX + "/" + ruleId);
+            request.AddHeader("Content-Type", "application/json");
+            request.SetContent(JsonConvert.SerializeObject(ruleRequest));
+
+            var response = this.httpClient.PutAsync(request).Result;
+
+            // Dispose after tests run
+            this.disposeRulesList.Add(ruleId);
+
+            // Act
+            request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX + "/" + ruleId);
+
+            response = this.httpClient.DeleteAsync(request).Result;
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        }
+
+        private RuleApiModel GetSampleRuleWithCalculation(string calculation, string timePeriod)
+        {
+            var condition = new ConditionApiModel()
+            {
+                Field = "pressure",
+                Operator = "GreaterThan",
+                Value = "150"
+            };
+
+            var conditions = new List<ConditionApiModel> { condition };
+
+            return new RuleApiModel()
+            {
+                Name = calculation + " Test Rule",
+                Description = "Test Description",
+                GroupId = DEFAULT_CHILLERS_GROUP_ID,
+                Severity = "Info",
```

Comment:
I would recommend testing with different "Severity" by adding another argument to `GetSampleRuleWithCalculation`. No need to add new test. Just feeding the existing test with different severity levels will do.

Reply:
Good to have. @jillcary Can you do that as a separate check-in. I'll approve and merge this one.